### ISSUE TITLE
Consider wallpaper colour / Power BI Apps

### ISSUE THREAD

**Issue by infojamjames** [andymarsden/pbi-templates#12]
Colours may not make the canvas look like the main focus

**Comment by sylwiasajdakLCC** [comment]
Example of what the background looks like when published in an app

**Comment by infojamjames** [comment]
In my and Andy's opinion the wallpaper should not be changed. If there is an accessibility issue the Power BI Apps borders should be changed.

**Comment by infojamjames** [comment]
Feel free to test and change the colour of the wallpaper of the template, I guess there is only a limited choice of colours.

**Comment by infojamjames** [comment]
Wallpaper colour changed

**Comment by sylwiasajdakLCC** [comment]
Forgot to mention in the call, but this is what the report looks like with a lighter wallpaper and with a matching app header:
![image](https://github.com/andymarsden/pbi-templates/assets/143817660/a05e1099-b3fe-4f8d-a069-bac29dba1a9c)

**Comment by infojamjames** [comment]
Just thinking, do you reckon people would be just using the default app border though @sylwiasajdakLCC ? 
![image](https://github.com/andymarsden/pbi-templates/assets/87974094/3f6a8de0-beb2-465e-bbaa-0aff597884ba)
